authortechnical.Rmd

```diff
@@ -307,11 +310,11 @@ To get the citation for an article in [Google Scholar](https://scholar.google.co
 knitr::include_graphics("images/citation-gscholar.png")
 ```
 
-### Embedded tweets {#addtweet}
+### Embedded social media post {#addsocialpost}
 
-Use a [Hugo shortcode](https://gohugo.io/content-management/shortcodes/#tweet) to embed a tweet using its username and its ID e.g. `{{{< tweet user="SanDiegoZoo" id="1453110110599868418" >}}}`. In R Markdown, shortcodes need to be html escaped, refer to [the template](https://github.com/ropensci/roweb3/blob/HEAD/archetypes/Rmd/index.md) for an example.
+You can get the code to embed a social media post by clicking on the post's timestamp and then on **"Embed"** for most of the social media platforms.  The social media platform will provide a code snippet that you can copy and paste into your post.
```

Comment:
should we say we don't want embedded x posts?

Reply:
Good question.

Reply:
So, after thinking a while, I'd actually go for a hard rule (but won't veto your decision!!). I'd even go as far as suggesting we try not to rely on embedded "anything" a lot or at least not in a way that's necessary for the post. 

This thought is brought to you by my having to delete lines from posts that linked to deleted posts :wink:

I'm also re-reading https://gohugo.io/about/privacy/#disable-all-services :smile_cat:

Reply:
In the guide, we already show an alternative way to cite something someone says on social media as a quote: adding the link to the post. This way, people can still use those quotes without embedded content. In any case, if the link to the post is deleted, it will be a 404 in the future.  But it will work for the history. 

We can remove all the embedded instructions and explicitly say we prefer to quote in this other way and not by embedding content. 

If we can avoid the work of going back and updating old blog posts because something has changed in social media, we should take that chance.